config/locales/views.es.yml

```
@@ -169,6 +169,27 @@ es:
       create:
         from_comment: 'De comentario por %{user} el %{date}'
 
+  licenses:
+    general_payment_reason: 'Pago mensual MawidaBP'
+    trial_valid_until: 'Período prueba valida hasta'
+    price_per_month: 'Precio por mes'
+    statuses:
+      unpaid: 'Impago'
+      active: 'Activo'
+      trial: 'Prueba'
+      cancelled: 'Cancelado'
+    blocked:
+      show:
+        body: 'Su organización está inhabilitada por falta de pago'
+        check: 'Volver a comprobar pago'
+    update:
+      correctly_created: 'Suscripción creada correctamente'
+    show:
+      check: 'Volver a comprobar pago'
+    paypal_button:
+      create: 'Suscribirse'
+      create_error: 'Ha ocurrido un problema. Si se efectuó el pago, por favor comuniquese con nosotros.'
```

```suggestion
      create_error: 'Ha ocurrido un problema, por favor comuníquese con soporte@mawidabp.com para comprobar si el pago está acreditado.'
```

Igual, con un `_html` y demás.